packages/web/src/shared/useLocalStorageState.ts

```diff
@@ -0,0 +1,25 @@
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
+
+export default function useLocalStorageState<
```

Comment:
This hook works like react's `useState`, but it will save the current value in local storage so it persists on refresh